### PR TITLE
volume module sample playbook

### PR DESCRIPTION
**Pull request by ketankelkar**
add volume initialization sample playbook which leverages the zos_ickdsf_init module

Signed-off-by: Ketan Kelkar <ktnklkr@gmail.com>

### CODE REVIEW

**Comment by ketankelkar** [comment]
an idea from a discussion with @dan-jast and myself was to have a scenario available where a volume is initialized and a ZFS mount is set up (leverage zos_mount) -- i see two options, either modify the existing playbook to set up a file system and copy over JCL to a Unix Systems Services file (rather than the current copy to a member) and submit the job from that environment. Alternatively, add a parallel playbook that sets up the volume, the ZFS mount and run JCL there

File: zos_concepts/volume_management/volume_initialization/init_dasd_vol_and_run_sample_jcl/files/HELLO.jcl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//* Notes:
+//*   If you change the job name, you must update the playbook job name
+//*   references.
+//******************************************************************************
+//HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',
+//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID
+//*
+//* PRINT "HELLO WORLD" ON JOB OUTPUT
+//*
+//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
```
Comment (ddimatos) [inline zos_concepts/volume_management/volume_initialization/init_dasd_vol_and_run_sample_jcl/files/HELLO.jcl:22]: This is an irrelevant old comment probably when doing some negative testing. Probably don't need that. 
//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
AND WILL CAUSE A JCL ERROR

File: zos_concepts/volume_management/volume_initialization/init_dasd_vol_and_run_sample_jcl/inventories/group_vars/all.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  _BPXK_AUTOCVT: "ON"
+  ZOAU_HOME: "{{ ZOAU }}"
+  PYTHONPATH: "{{ ZOAU }}/lib"
+  LIBPATH: "{{ ZOAU }}/lib:{{ PYZ }}/lib:/lib:/usr/lib:."
+  PATH: "{{ ZOAU }}/bin:{{ PYZ }}/bin:/bin:/var/bin"
+  _CEE_RUNOPTS: "FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+  _TAG_REDIR_ERR: "txt"
+  _TAG_REDIR_IN: "txt"
+  _TAG_REDIR_OUT: "txt"
+  LANG: "C"
```
Comment (ddimatos) [inline zos_concepts/volume_management/volume_initialization/init_dasd_vol_and_run_sample_jcl/inventories/group_vars/all.yml:19]: We should add the new environment variable here and describe it below.

**Review by ddimatos — CHANGES_REQUESTED** [review]
2 small comments, otherwise this looks great, thanks @ketankelkar

**Review by ddimatos — APPROVED** [review]
Thanks for the updates, looks good @ketankelkar

**Pull request merged**